data/json/mapgen/lake_buildings/lighthouse.json

```diff
@@ -32,7 +32,7 @@
         "****~~**~~~//~~~~~~*****"
       ],
       "place_vehicles": [
-        { "vehicle": "boat_motor_single", "x": 4, "y": 5, "rotation": 180, "chance": 10, "status": 3 },
```

Comment:
It's not possible for a 11x7 boat to spawn here.  It won't fit within the existing map.  If it tries to overlap to another OMT, it won't spawn at all.

If you want to add it here, you need to add a 2nd OMT to the special and then you'll have space to spawn it.  I'd place it to the north of the dock.

Reply:
Fixed by adding a blank 'lake' tile to the overmap and spawning the lake vehicle group there, the boat_motor_single was re-added